### ISSUE TITLE
Upgrade material-ui to version 1 

### ISSUE THREAD

**Issue by devshawn** [devshawn/password.live#17]
This may not be a super easy task, but since I wrote this a year ago, material UI v1 has been released. We should update the client to latest version of material ui. 

This may require changing how styling works or which components are used since there is breaking changes; ideally, we'd keep the design as similar as possible to the current design.

**Comment by seleregb** [comment]
Hi, I would like to work on this for you. 

The latest version of material UI is v3.3.1 though so is it that version or v1 specifically?.

**Comment by devshawn** [comment]
Awesome, @seleregb! The latest version of material UI would be ideal.

**Comment by kndarp** [comment]
Hi! I'd like to contribute to this issue

**Comment by seleregb** [comment]
@devshawn I'm almost done, but I just noticed that you've updated the master branch again, as I'm using the live site as my guide.

**Comment by devshawn** [comment]
Correct, @seleregb -- we've made a lot of changes while working on it. You'll have to merge the latest master and make your changes accordingly -- there was a lot of changes in a short period -- sorry :)

**Comment by devshawn** [comment]
(The live site has also been updated now)

**Comment by seleregb** [comment]
@devshawn I'm having issues at the moment with `react-hot-loader` since my merge with master

> ERROR in ../server/constants/password.constants.js
Module not found: Error: Can't resolve 'react-hot-loader' in 'C:\Users\186866380\Documents\Source\Repos\password.live\source\server\constants'
 @ ../server/constants/password.constants.js 2:20-47 19:23-50 21:20-47
 @ ./application/settings/settings.reducer.js
 @ ./application/reducers.js
 @ ./application/store.js
 @ ./client.js
 @ multi (webpack)-dev-server/client?http://localhost:5500 webpack/hot/dev-server @babel/polyfill react-hot-loader/patch ./client

**Comment by devshawn** [comment]
I just pushed code to fix that, someone's PR broke that.

**Comment by seleregb** [comment]
Good to know. Thanks for fixing that

**Comment by seleregb** [comment]
Hey @devshawn , I'm thinking of taking this up again for this years Hacktoberfest. I feel like there will need to be massive library updates to reach the promised land  😅
